### PR TITLE
NOT QUITE FROZEN BUT STILL SORTA CHILLY

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
We're gonna try something else since everyone is making features anyway

For every dumb meme balance PR you make, you must make one fix PR.

This one is indefinite.

Yes it's probably exploitable, but it's better than everyone just waiting a month for the freeze to end or bypassing it anyway.

We have 100 PRs a week, so even if everyone just fixes something tiny for every one of those PRs made, it should still be a substantial amount of stuff getting fixed.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
Brr.

**Comment by Kromgar** [comment]
A rebellion is brewing

**Comment by PKPenguin321** [comment]
>ive already made 2 fix PRs

yeee

**Comment by coiax** [comment]
As someone who's actually been fixing shit this freeze, I'm not in favour of this.

**Comment by xxalpha** [comment]
Finally you're using your powers for good.

**Comment by MMMiracles** [comment]
i fixed like 2-3 issues that weren't even REPORTED, that's how good i am

**Comment by lzimann** [comment]
how hard is to people stop with balance/feature for a month??

**Comment by GunHog** [comment]
> how hard is to people stop with balance/feature for a month??

The freeze had no prizes. No incentives. There was very little warning of its coming as well. It is easy to stop people making balance/feature PRs by banning them or closing the PRs. It is hard to get unpaid volunteers to make only bugfixes when you do not offer them anything.

It was doomed to fail, as no one liked or respected it. It had very little enforcement. It was even violated on so many occasions that it may as well have not existed at all.

**Comment by Dorsisdwarf** [comment]
_ALL_ our freezes fail to hit targets, even when we start them right before phil's bi-seasonal Bug Genocides. This includes when there was like $70 of vidya games up for grabs too.

There's just so many bugs and they're added so fast and are frequently so incomprehensible

**Comment by KorPhaeron** [comment]
goof downvoting this is how I know its a good idea

File: code/__DEFINES/math.dm [hunk 1]
```diff
@@ -8,4 +8,4 @@
 #define ONE_ATMOSPHERE			101.325	//kPa
 #define T0C						273.15	// 0degC
 #define T20C					293.15	// 20degC
-#define TCMB					2.7		// -270.3degC
+#define TCMB					really cold		// -270.3degC
```
Comment (PKPenguin321) [inline code/__DEFINES/math.dm]: only kinda cold imo

**Comment by ike709** [comment]
One bug fix per one meme PR? We'll have 0 bugs by the end of the week if this could actually be enforced.

**Comment by Bawhoppen** [comment]
give coiax his free permaban kor its only fair

**Comment by JJRcop** [comment]
@ike709 It can be enforced: your PR can't be merged unless you have a fix PR ready to merge

**Comment by ike709** [comment]
Yeah, just make people link their fix PR in their feature/meme OP.

**Comment by optimumtact** [comment]
I think this is pretty embedded in now

**Comment by JJRcop** [comment]
I think we should have a "used token" tag for now

**Pull request closed without merging**